content/updates/serenity-updates-for-the-pim.json

```diff
@@ -0,0 +1,158 @@
+{
+    "data": [
+        {
+            "startDate": "June, 4th 2020",
+            "notification_duration": 14, 
```

Comment:
Should it have to be added in the json even if it's not used in the UI until the step 4 is done ? If yes, could we name it in camel case to be consistent with the other properties like 'startDate' and 'altImg' ?

Reply:
Yes, I'll rename this property. And regarding the fact that it's not used before step 4, you're totally right. If it's not more work to have it already in the json file, let's keep it.